src/MICore/NativeMethods.cs

```diff
@@ -8,15 +8,15 @@ namespace MICore
 {
     internal class NativeMethods
     {
-        // TODO: It would be better to route these to the correct .so files directly rather than pasing through system.native. 
+        private const string Libc = "libc";
```

Comment:
Would it be best to make the pinvokes internal and write wrapper methods that verified the platform?

From: Wiktor Kopec [mailto:notifications@github.com]
Sent: Friday, February 5, 2016 1:49 PM
To: Microsoft/MIEngine <MIEngine@noreply.github.com>
Cc: Jackson Davis <Jackson.Davis@microsoft.com>
Subject: Re: [MIEngine] Changing entry points for System.Native calls. (#171)


In src/MICore/NativeMethods.cs<https://github.com/Microsoft/MIEngine/pull/171#discussion_r52077328>:

> @@ -8,15 +8,15 @@ namespace MICore

>  {

>      internal class NativeMethods

>      {

> -        // TODO: It would be better to route these to the correct .so files directly rather than pasing through system.native.

> +        private const string Libc = "libc";

These functions are called by LocalLinuxTransport, which is only used after checking the platform. I will rename the class.

—
Reply to this email directly or view it on GitHub<https://github.com/Microsoft/MIEngine/pull/171/files#r52077328>.